client/components/my-plan-card/README.md

```diff
@@ -0,0 +1,42 @@
+My Plan Card
```

Comment:
I'd like to note that this component doesn't make much sense in the global components library - I don't see how it's useful as a global component. It makes sense only in the plans context, so maybe it makes most sense to live there instead. What do you think?

Reply:
I think it makes sense to move it to `my-sites/plans` directory. It would replace the `<CurrentPlanHeader />` component that's currently used there and intended to be replaced by this new component in #37642.

Reply:
Let's do that 😉 Does it need an Asana card?